r_basics_visualize_data/r_basics_visualize_data.md

```diff
@@ -0,0 +1,843 @@
+
+![survminer cheat sheet](media/survminer.png)<!-- style = "max-width:700px;" -->
+
+### A Grammar for Tables
+
+![hex icons for gt and gtsummary](media/gt_icons.png)<!-- style = "max-width:300px;" -->
+
+![attractive data table with subgroups and footnotes](media/gt_table.png)<!-- style = "max-width:500px;" -->
+
+The `gt` package provides a grammar for creating display tables, i.e. tables that you might want to show in a publication or on a summary report. The `gtsummary` package makes it trivial to generate publication-ready tables from a tidy data frame.
+
+## Recap
+
+| ![ggplot2 logo on a hexagon.  The logo consists of the word "ggplot2" superimposed on light grey graph paper with a line graph connecting blue dots of different shades](media/ggplot2_hex.png)<!-- style = "max-width:200px;" --> | **ggplot2** is a package that provides a **grammar of graphics**. You can create any type of plot using a simple template to which you provide: |
+| ![Table with rows and columns.  The data is not visible, although headers are, and include mrn, gender, `test_id`, and result. In each row and column, an arrow spans the entire row or entire column.](media/tidy_data.png)<!-- style = "max-width:200px;" --> | 1. A **tidy data frame**, in which each variable is in its own column, each observation is in its own row, and each value is in its own cell; |
+| ![histogram of covid data](media/mini_histogram.png)<!-- style = "max-width:75px;" -->  ![frequency polygon of covid data](media/mini_freqpoly.png)<!-- style = "max-width:75px;" --> | 2. A **geom function**, which tells R what kind of plot to make; and |
+| ![Crossed X and Y axes](media/positions.png)<!-- style = "max-width:75px;" -->  ![Display of color choices](media/colors.png)<!-- style = "max-width:75px;" -->  | 3. **Aesthetic mappings**, which tell R how to represent data as graphical markings on the plot. |
+| ![Histogram of Covid tests as a function of `pan_day`.  The bars forming the histogram have three colors: a small blue section at the bottom, representing positive results, a much larger green section in the middle for negative results, and a tiny red section at the top for invalid test results.](media/multicolor_histogram.png)<!-- style = "max-width:75px;" -->   ![Histogram of Covid tests as a function of `pan_day`.  The bars forming the histogram have three colors: a small blue section at the bottom, representing positive results, a much larger green section in the middle for negative results, and a tiny red section at the top for invalid test results.](media/blue_histogram.png)<!-- style = "max-width:75px;" -->   | Aesthetics can be **mapped** to a variable or **set** to a constant value. |
+| | Additionally, you can do things like change theme, color palette, coordinate systems, facet your graph, and more! |
+
+## Additional Resources
+
+![Book cover of Fundamentals of Data Visualization](media/fundamentals.png)<!-- style = "max-width:400px;" -->

```

Comment:
```suggestion
![""](media/fundamentals.png)<!-- style = "max-width:400px;" -->

```

doesn't really add anything useful to the experience of the person with limited visual access